backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHSparkPlanExecApi.scala

```diff
@@ -370,4 +378,103 @@ class CHSparkPlanExecApi extends SparkPlanExecApi {
     new CHSha1Transformer(substraitExprName, child, original)
   }
 
+  /**
+   * Generate an BasicScanExecTransformer to transfrom hive table scan. Currently only for CH
+   * backend.
+   * @param child
+   * @return
+   */
+  override def genHiveTableScanExecTransformer(child: SparkPlan): BasicScanExecTransformer = {
+    if (!child.getClass.getSimpleName.equals("HiveTableScanExec")) {
```

Comment:
use 'isInstanceOf' ?

Reply:
HiveTableScanExec is a private class, it can not be used like child.isInstanceOf[HiveTableScanExec]

Reply:
@KevinyhZou 
HiveTableScanExec is private in the "hive" package. It can be used in the hive package to do some transform.
Others, BatchScanExec is extended with DS v2, and FileScanExec is extended with DS v1. My suggestion is that HiveTableScanExec should use it's own transform like hiveexectransform. hiveexectransform can be trait with BasicScanExecTransformer

Reply:
OK, it will be better to implements this to put the logic into a HiveExecTransformer. And I will try to do this. thanks @loneylee